src/node.rs

```diff
@@ -336,6 +346,38 @@ impl Node {
             .unwrap_or_default()
     }
 
+    /// Returns a set of elders we should be connected to.
+    pub fn elders(&self) -> impl Iterator<Item = &PublicId> {
+        self.chain().into_iter().flat_map(Chain::elders)
+    }
+
+    /// Returns their knowledge
+    pub fn get_their_knowledge(&self) -> BTreeMap<Prefix<XorName>, u64> {
+        self.chain()
+            .map(Chain::get_their_knowledge)
+            .cloned()
+            .unwrap_or_default()
+    }
+
+    /// If our section is the closest one to `name`, returns all names in our section *including
+    /// ours*, otherwise returns `None`.
+    pub fn close_names(&self, name: &XorName) -> Option<Vec<XorName>> {
+        self.chain().and_then(|chain| chain.close_names(name))
+    }
+
+    /// Only if we have a chain (meaning we are elders) we will process this API
+    pub fn min_sec_size_from_chain(&self) -> Option<usize> {
+        self.chain().map(Chain::min_sec_size)
+    }
```

Comment:
Again, there already is `Node::min_section_size`. How is this different and why do we need this?

Reply:
Hmm so `Node::min_section_size` appears, unused, should we just remove that one instead?

Reply:
Added a commit where I remove the unused `Node::min_section_size` and only use the chain backed version